src/Pretzel.Logic/Commands/ICommandParameters.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+
+namespace Pretzel.Logic.Commands
+{
+    public interface ICommandParameters
```

Comment:
Could you add some comments on public api?

Reply:
Done.